### ISSUE TITLE
Myocardial pyrophosphate uptake

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9860]
**Preferred term label:**
Myocardial pyrophosphate uptake

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cardiac amyloidosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000067
- label: Increased myocardial pyrophosphate uptake
- parents: [Abnormal myocardium morphology (HP:0001637)](https://hpo.jax.org/app/browse/term/HP:0001637)
- pmid: PMID:22149423
- def: Amount of pyrophosphate uptake by the myocardium as demonstrated by by technetium pyrophosphate (99mTc-PYP) scintigraphy above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9860
- synonyms:
